src/components/ReviewSummary/TokenReviewSummary.js

```diff
@@ -61,7 +61,8 @@ export const TokenReviewSummaryComponent = ({
 
   if (asset) {
     assetIcon = { uri: `${getEnv().SDK_PROVIDER}/${asset.iconUrl}?size=3` };
-    fiatAmount = fiatAmount || getFormattedRate(rates, amount, asset.symbol, baseFiatCurrency || defaultFiatCurrency);
+    fiatAmount =
+      fiatAmount || getFormattedRate(rates, amount, asset.symbol, baseFiatCurrency || defaultFiatCurrency, true);
```

Comment:
I know it wasn't in this PR scope, but double different ternary here in single line little confusing, maybe this is easier, wdyt?
```suggestion
    if (!fiatAmount) {
      fiatAmount = getFormattedRate(rates, amount, asset.symbol, baseFiatCurrency || defaultFiatCurrency, true);
    }
```

Reply:
@poocart It make sense. I will do the changes.